index.bs

```diff
@@ -5977,6 +5971,9 @@ The {{float}} type is a floating point numeric
 type that corresponds to the set of finite single-precision 32 bit
 IEEE 754 floating point numbers. [[!IEEE-754]]
 
+When defined with the [{{Unrestricted}}] [=extended attribute=], the {{float}} type
+additionally corresponds to IEEE 754 non-finite, and special "not a number" values (NaNs). [[!IEEE-754]]
```

Comment:
IEEE 754 specifies several versions of non-finite and not-a-number values: single-precision, double-precision, etc. You should say single-precision 32-bit to be consistent with above.